### DISCUSSION TITLE
Can't install 2.6.0 from IDE 1.8.13

### DISCUSSION THREAD

**Discussion by benjavita** [SpenceKonde/megaTinyCore#813] [Library discussion]
Hi, I'm getting the attached error, can you please check the download url is correct?

Cheers
![image](https://user-images.githubusercontent.com/50239136/190615536-4849d9fb-de07-4426-8e8f-63577f639b7d.png)

**Comment by SpenceKonde** [comment]
Read the text. 2.6.0 was a disaster, you couldn't compile anything, and it didn't matter if you could because if you were on windows, the linker wouldn't work, and the the upload recipe in platform.txt got corrupted late in the development process from a bungled merge. 

This is doing EXACTLY what it is supposed to do: PREVENT YOU FROM INSTALLING IT. 

Once a board manager version has been made available, that version must never be removed, otherwise bad things happen to people who have installed it. I took some heat for trying to do that earlier in the year for much the same reason (a critical bug was reported just hours after the release (and it wasn't usable for the majority of applications), and I thought I could just yank the release from board manager. 

So when 2.6.0 turned out to be garbage, all I could do is tell people not to use it, and delete all the defective files so that it couldn't be installed. I withdrew the version from github release too, you'll notice.

This is the current changelog for the 2.6.1 version that I'm working on currently:
* Bugfix: Correct several USART bugs that snuck through, including a regression that would prevent proper serial operation.
* Bugfix: Azduino5 was no good, no sketches would compile. Same filesystem structure error as usual, moved to Azduino6.
* Bugfix: The 4000 lines of workarounds for compatibility issues introduced by the toolchain update didn't make it into the megaTinyCore release.
* Bugfix: The platform.txt file had been trashed by a bungled merge and tried to upload based on params that didn't exist.
* Bugfix: Correct an include path that confused some alternative IDEs.
* Bugfix: Correct compile error involving RS485 USART configuration.
* Bugfix: Comparator library namespace error fixed. (thanks @ObviousInRetrospect)
* Bugfix: Azduino5 and initial Azduino6 package for windows (but not other platforms) used the bunk ldscripts and would fail at link time. 
 
Thankfully Azduino6 windows toolchain package  can be updated since it hasn't been installed automatically by any released version.

And right now I'm working on a newly discovered issue in boards.txt, so 2.6.1 is not ready for release.

**Reply by ObviousInRetrospect** [reply]
why does the text say to use 2.5.9? Is there a problem with 2.5.11?

**Comment by SpenceKonde** [comment]
No, it's because I was tired AF when I wrote it.
